stubs/CoreGenericFunctions.phpstub

```diff
@@ -1086,6 +1086,31 @@ function chunk_split(string $str, int $chunklen = 76, string $end= ''): string
 {
 }
 
+/**
+ * @template TInput as \Countable|\SimpleXMLElement|\ResourceBundle|array
+ *
+ * @param TInput $value
+ * @param 0|1 $mode
+ *
+ * @return (
```

Comment:
Also to be handled: `non-empty-countable` type, which I discovered somewhere in tests, and didn't know of before :D